src/style.css

```
@@ -0,0 +1,143 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  width: 300px;
+  text-align: center;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+}
+
+.block--1 {
+  background-color: red;
+}
+
+.block--2 {
+  background-color: rgb(204,0,0);
+}
+
+.block--3 {
+  background-color: rgb(153,0,0);
+}
+
+.block--4 {
+  background-color: rgb(102,0,0);
+}
+
+.block--5 {
+  background-color: rgb(51,0,0);
+}
+
+.block--6 {
+  background-color: black;
+}
+
+@media screen and (min-width: 300px) and (max-width: 599px) {
```

```suggestion
@media screen and (min-width: 300px){
```
Нижний медиа запрос перебьет этот